stix_shifter_modules/trustar/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,251 @@
+{
+  "IP": {
+    "key": "ipv4-addr.value",
+    "object": "ipv4-addr"
+  },
+  "MALWARE": {
+    "key": "x-trustar-malware.value",
+    "object": "x-trustar-malware"
+  },
+  "URL": {
+    "key": "url.value",
+    "object": "url"
+  },
+  "CVE": {
+    "key": "x-trustar-vulnerability.CVE",
+    "object": "x-trustar-vulnerability"
+  },
+  "EMAIL_ADDRESS": {
+    "key": "email-addr.value",
+    "object": "email-addr"
+  },
+  "REGISTRY_KEY": {
+    "key": "windows-registry-key.id",
+    "object": "registry-key"
+  },
+  "MD5": {
+    "key": "file.hashes.MD5",
+    "object": "file"
+  },
+  "SHA1": {
+    "key": "file.hashes.SHA1",
+    "object": "file"
+  },
+  "SHA256": {
+    "key": "file.hashes.SHA-256",
+    "object": "file"
+  },
+  "SOFTWARE": [{
+      "key": "software.name",
+      "object": "file"
```

Comment:
this is wrong. object should be something different not "file" that will create a separate software object